taskcluster/kinds/firefoxci-artifact/kind.yml

```diff
@@ -6,10 +6,12 @@ loader: taskgraph.loader.transform:loader
 
 transforms:
   - fxci_config_taskgraph.transforms.firefoxci_artifact
+  - taskgraph.transforms.cached_tasks
   - taskgraph.transforms.run
   - taskgraph.transforms.task
 
 task-defaults:
+  attributes: {}  # cached_tasks transforms assume this exists
```

Comment:
That's...annoying. Consider fixing this upstream if you feel like it? (That does not block this landing of course.)